leaderboardFeature/src/main/java/com/panhuk/leaderboardfeature/utils/Image.kt

```diff
@@ -0,0 +1,21 @@
+package com.panhuk.leaderboardfeature.utils
+
+import com.panhuk.leaderboardfeature.R.drawable
+import kotlin.random.Random
+
+fun getDrawable(): Int {
```

Comment:
might be better to put all these items into list and return element by generated random index
This way we won't have any magic numbers and long when case